packages/communication-react/stable-review/communication-react.api.md

```diff
@@ -144,7 +144,7 @@ export type AzureCommunicationCallWithChatAdapterFromClientArgs = {
 // @public
 export type AzureCommunicationChatAdapterArgs = {
     endpoint: string;
-    userId: CommunicationUserIdentifier;
+    userIdentifier: CommunicationUserIdentifier;
```

Comment:
we unfortunately cannot change this here as it will break the stable API. I think it's best to constrain these changes to storybook only

Reply:
ah okay this was something I was wondering, this seemed like a pretty major change for a bug.